core/services/ocr2/plugins/ocr2keeper/evm21/logprovider/integration_test.go

```diff
@@ -660,6 +653,7 @@ func setupDependencies(t *testing.T, db *sqlx.DB, backend *backends.SimulatedBac
 	pollerLggr.SetLogLevel(zapcore.WarnLevel)
 	lorm := logpoller.NewORM(big.NewInt(1337), db, pollerLggr, pgtest.NewQConfig(false))
 	lp := logpoller.NewLogPoller(lorm, ethClient, pollerLggr, 100*time.Millisecond, false, 1, 2, 2, 1000)
+	require.NoError(t, lp.Start(testutils.Context(t)))
```

Comment:
None of the tests calling this `setupDependencies` helper had a running log poller. Was that intentional?